### DISCUSSION TITLE
Checkmark icon in the Checkbox component

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#649] [Q&A] [answered]
I discovered that the checkmark icon in the Checkbox component shows it as stroke icon instead filled when you zoom 110% and above. Is this intentional? Please see the attached link for the detail. [https://drive.google.com/file/d/1UMWQxBR3h0NaOa9bh_RTIQ1sjgl4DraQ/view?usp=sharing](url)

**Comment by SiTaggart** [comment, marked as answer]
Hi @sang-yeo 

It's not intended but is a known issue with the current implementation. We'll be replacing it with a real icon in the future so it doesn't suffer as much degradation on scaling. Thanks for the comment 🙏

**Reply by sang-yeo** [reply]
Thanks for the quick reply. I will sync with my dev. 🥂
